src/domain/ticketSystemPassword/api/ticketSystemPasswordsApi.ts

```diff
@@ -0,0 +1,44 @@
+import { Event } from '../../../api/generatedTypes/Event';
+import { MethodHandlerParams } from '../../../api/types';
+import {
+  handleApiNode,
+  mapLocalDataToApiData,
+  mutationHandler,
+} from '../../../api/utils/apiUtils';
+import { importEventTicketSystemPasswordsMutation } from '../mutations/TicketSystemPasswordMutations';
+
+export type ErrorType = {
+  field: string;
+  message: string;
+  value: string;
+};
+
+// TODO: Generate
```

Comment:
What about this one? 🙂

Reply:
actually, it is. I think those should be generated somehow automatically.